public/app/plugins/datasource/postgres/PostgresResponseParser.ts

```diff
@@ -0,0 +1,28 @@
+import { uniqBy } from 'lodash';
+
+import { DataFrame, MetricFindValue } from '@grafana/data';
+import { ResponseParser } from 'app/features/plugins/sql/types';
+
+export default class PostgresResponseParser implements ResponseParser {
+  transformMetricFindResponse(frame: DataFrame): MetricFindValue[] {
+    const values: MetricFindValue[] = [];
+    const textField = frame.fields.find((f) => f.name === '__text');
```

Comment:
looks like this might be the same as mysql.  maybe we can create a core ResponseParser that could be extended